website/content/docs/job-specification/disconnect.mdx

```diff
@@ -0,0 +1,202 @@
+---
+layout: docs
+page_title: disconnect Block - Job Specification
+description: |-
+  The "disconnect" block describes the behaviuor of both the Nomad server and 
+  client in case of a network partition, as well as how to reconcile the workloads
+  in case of a reconnection.
+---
+
+# `disconnect` Block
+
+<Placement groups={['job', 'group', 'disconnect']} />
+
+The `disconnect` block describes the systm's behaviour in case of a network 
+partition.
+
+```hcl
+    job "docs" {
+  	    group "example" {
+    	    disconnect {
+		        lost_after = "6h"
+		        stop_after = "2h"
+		        replace = false
+                reconcile = "keep_original"
+     	    }
+  	    }
+    }
+
+```
+
+## `disconnect` Parameters
+
+- `lost_after` `(string: "")` - Specifies a duration during which a Nomad client
+  will attempt to reconnect allocations after it fails to heartbeat
+  in the [`heartbeat_grace`] window.  It defaults to “” which is equivalent to 
+  having the disconnect block be nil.
+  
+  See [the example code below][lost_after] for more details. This setting cannot
+  be used with [`stop_after`].
+
+- `replace` `(bool: false)` - Specifies if the disconnected allocation should 
+  be replaced by a new one rescheduled on a different node. If false and the 
+  node it is running on becomes disconnected or goes down, this allocations
+  wont be rescheduled and will show up as `unknown` until the node comes back 
+  up or it are manually stoped:
+
+  ```plaintext
+  `nomad alloc stop  <alloc ID>`
+  ```
+
+  If true, a new  alloc will be placed immediately upon the node becoming 
+  disconnected.
+
+- `stop_after` `(string: "")` - Specifies a duration after which a disconnected 
+  Nomad client will stop its allocations. Keep in mind that if stop_after is 
+  shorter than ttl, and `replace` = false, can lead to a state where no allocations
+  are running, hence it is not admitted and will cause a validation error.
```

Comment:
We're describing the "why" before the "what" here. I'd reword this to make it clear at the start of the sentence that the configuration described isn't allowed, and _then_ describe why. Also, I'm not sure what "ttl" we're referring to here?

```suggestion
  Nomad client will stop its allocations. Setting `stop_after` shorter than 
  ??? and `replace = false` at the same time is not permitted and will cause a 
  validation error, because this would lead to a state where no allocations 
  can be scheduled.
```